src/common/Navbar.jsx

```diff
@@ -1,81 +1,142 @@
-           
-            <Tabs 
-                className={classes.navlinks}
-                value={value} 
-                onChange={handleChange} 
-                aria-label="simple tabs example" 
-            >
-                {
-                    navItems.map((el, i) => (
-                        <Tab 
-                            key={i}
-                            className={classes.navlink}  
-                            label={el} 
-                            onClick={() => console.log(`${el}`)}
-                        />
-                        )
-                    )
-                }
-            </Tabs>
+        <AppBar className={classes.root}>
+            <Toolbar className={classes.content}>
+                <Link href="/" className={classes.logo}>
+                    <Image src={logo} width={164}/>
+                </Link>
+            
+                <Hidden mdDown>
+                    <Box className={classes.navlinks} aria-label={`navbar `}>
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
                    <Box className={classes.navlinks} aria-label={"navbar "}>
```